Dockerfile

```
@@ -27,10 +28,10 @@ RUN apt-get update && apt-get install -y \
 	qtcreator
 
 COPY . /root/mumble
-WORKDIR /root/mumble
+WORKDIR /root/mumble/release
```

```suggestion
WORKDIR /root/mumble/build
```

`build` is the usual name for the build-dir used for cmake. In the end this doesn't really matter though :shrug:

---

I used `release` because it was used previously in Dockerfile. I changed it to `build`.